packages/ui-design-system/src/Menu/Menu.stories.tsx

```diff
@@ -13,7 +13,7 @@ import {
   MenuItem,
   MenuPopover,
   MenuRoot,
-} from './MenuWithCombobox';
+} from './Menu';
```

Comment:
Rename since this was confusing: you can actually use this without a combobox, I started by implementing a new component before figuring out it was already done...